auto-layout-stencil.shader

```diff
@@ -469,7 +469,7 @@ float4 mainImage(VertData v_in) : TARGET
     } else if (isBlack(r) && isGrey(g) && isGrey(b)) { //title screen
         return renderTitle(uv);
     } else if (isGrey(r) && isGrey(g) && isBlack(b)) { //level-select / high-score
-        float4 o = sampleBlock(orange_box(), pixelSize);
+        float4 o = sampleBlock(orange_uv(), pixelSize);
```

Comment:
I'm not sure if this is proper. I only took care of fixing the types (sampleBlock() takes 2 float2 values but orange_box() returns a float4).